server/src/db/mongo-connect.ts

```diff
@@ -0,0 +1,13 @@
+import mongoose from 'mongoose';
+import { envConfig } from '../config/env.config';
+
+export const mongoConnect = async () => {
+	try {
+		await mongoose.connect(envConfig.mongoUrl);
+		console.log('Connected to MongoDB');
+	} catch (error) {
+		console.log(error);
+	}
+};
+
+mongoConnect();
```

Comment:
```suggestion
main()
	.then(() => {
		const workSpace = envConfig.nodeEnv ?? 'development';
		console.log(`[${new Date().toISOString()}] [INFO]: MongoDB connected successfully in '${workSpace}' mode.`);
	})
	.catch((err) => {
		console.error(`[${new Date().toISOString()}] [ERROR]: Failed to connect to MongoDB: ${err.message}`);
	});

async function main() {
	try {
		console.log(`[${new Date().toISOString()}] [INFO]: Attempting to connect to MongoDB...`);
		await mongoose.connect(envConfig.mongoUrl);
	} catch (err) {
		throw err;
	}
}
```
better approach to log also attempted connection